examples/cmri_mpca/config.py

```diff
@@ -0,0 +1,77 @@
+"""
+Default configurations for CMR processing and classification
+"""
+
+import os
+
+import numpy as np
+import pydicom
+from yacs.config import CfgNode as CN
+
+# -----------------------------------------------------------------------------
+# Config definition
+# -----------------------------------------------------------------------------
+
+_C = CN()
+
+# -----------------------------------------------------------------------------
+# Dataset
+# -----------------------------------------------------------------------------
+_C.DATASET = CN()
+_C.DATASET.SOURCE = "https://github.com/pykale/data/raw/main/image_data/ShefPAH-179/SA_64x64.zip"
+_C.DATASET.ROOT = "../data"  # '/shared/tale2/Shared'
+_C.DATASET.IMG_DIR = "DICOM"
+_C.DATASET.FILENAME = "PAH_SA_64x64"
+_C.DATASET.FILE_FORAMT = "zip"
+_C.DATASET.LANDMARK_FILE = "landmarks.csv"
+_C.DATASET.MASK_FILE = "Mask/phase1.dcm"
+# ---------------------------------------------------------------------------- #
+# Image processing
+# ---------------------------------------------------------------------------- #
+_C.PROC = CN()
+_C.PROC.SCALE = 2
+
+# ---------------------------------------------------------------------------- #
+# MPCA Pipeline
+# ---------------------------------------------------------------------------- #
+_C.MPCA = CN()
+_C.MPCA.CLF = ""  # choices=['CDAN', 'CDAN-E', 'DANN']
```

Comment:
What does CLF stand for? It is not readable even to me. Do we have domain adaptation here? Are the choices still applicable?